src/makielayout/types.jl

```diff
@@ -66,13 +66,20 @@ that are multiples of pi, printed like "1π", "2π", etc.:
 ```
 MultiplesTicks(5, pi, "π")
 ```
+
+If `strip_zero == true`, then the resulting labels
+will be checked and any label that is a multiple of 0
+will be set to "0".
 """
 struct MultiplesTicks
     n_ideal::Int
     multiple::Float64
     suffix::String
+    strip_zero::Bool
 end
 
+MultiplesTicks(n_ideal, multiple, suffix) = MultiplesTicks(n_ideal, multiple, suffix, false)
```

Comment:
This should get a keyword arg with default. At some number of arguments it's clearer to use keywords for the less common options. Then a reader of code has an easier time than with just a true or false